src/main/java/uk/gov/hmcts/reform/prl/controllers/CallbackController.java

```diff
@@ -84,7 +85,7 @@ public class CallbackController {
     private final AllTabServiceImpl allTabsService;
     private final UserService userService;
     private final DocumentGenService documentGenService;
-
+    private final ServiceOfApplicationService serviceOfApplicationService;
```

Comment:
Do we need this ServiceOfApplicationService here?

Reply:
You're correct, moved the controllers to a separate class - I have removed this.